bigquery/spec/import_spec.rb

```diff
@@ -0,0 +1,64 @@
+# Copyright 2015 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "spec_helper"
+require_relative "../import"
+
+RSpec.describe "Import table data" do
+  before do
+    @sample = Samples::BigQuery::Import.new
+
+    # create a temporary dataset / table
+    require "gcloud"
```

Comment:
`require` statements should be at the top of the file and not inside methods/modules/etc.  We only put `require` statements inside of methods for snippet purposes.